packages/database/src/sync/constants.ts

```diff
@@ -11,3 +11,12 @@ export const COLUMNS_EXCLUDED_FROM_SYNC = [
 ];
 
 export const SYNC_LOOKUP_PENDING_UPDATE_FLAG = -1;
+
+export const SYNC_TICK_FLAGS = {
+  INCOMING_FROM_CENTRAL_SERVER: -1,
+  LAST_UPDATED_ELSEWHERE: -999,
+  LOOKUP_PENDING_UPDATE: -2,
+  OVERWRITE_WITH_CURRENT_TICK: 0,
+};
+
+export const SYNC_CHANGELOG_TO_FACILITY_FOR_THESE_TABLES = ['patient_program_registrations'];
```

Comment:
Oh woah, we've already restricted it to PPR syncing down only? 

I didn't realise that. But also, imo this is a good place for inversion of control. Throughout sync we define rules at the model level, rather than having centralised sets of rules.